applications/ecallmgr/src/ecallmgr_fs_channel.erl

```diff
@@ -766,37 +770,37 @@ props_to_update(Props) ->
-      ,{#channel.fetch_id, props:get_value(<<"Fetch-ID">>, CCVs)}
       ,{#channel.bridge_id, props:get_value(<<"Bridge-ID">>, CCVs, UUID)}
-      ,{#channel.reseller_id, props:get_value(<<"Reseller-ID">>, CCVs)}
-      ,{#channel.reseller_billing, props:get_value(<<"Reseller-Billing">>, CCVs)}
-      ,{#channel.precedence, kz_term:to_integer(props:get_value(<<"Precedence">>, CCVs, 5))}
-      ,{#channel.realm, props:get_value(<<"Realm">>, CCVs, get_realm(Props))}
-      ,{#channel.username, props:get_value(<<"Username">>, CCVs, get_username(Props))}
-      ,{#channel.import_moh, props:get_value(<<"variable_hold_music">>, Props) =:= 'undefined'}
-      ,{#channel.answered, props:get_value(<<"Answer-State">>, Props) =:= <<"answered">>}
-      ,{#channel.profile, props:get_value(<<"variable_sofia_profile_name">>, Props)}
+      ,{#channel.callflow_id, props:get_value(<<"CallFlow-ID">>, CCVs)}
+      ,{#channel.cavs, CAVs}
       ,{#channel.context, props:get_value(<<"Caller-Context">>, Props)}
+      ,{#channel.destination, props:get_value(<<"Caller-Destination-Number">>, Props)}
       ,{#channel.dialplan, props:get_value(<<"Caller-Dialplan">>, Props)}
-      ,{#channel.to_tag, props:get_value(<<"variable_sip_to_tag">>, Props)}
+      ,{#channel.direction, kzd_freeswitch:call_direction(Props)}
+      ,{#channel.fetch_id, props:get_value(<<"Fetch-ID">>, CCVs)}
       ,{#channel.from_tag, props:get_value(<<"variable_sip_from_tag">>, Props)}
+      ,{#channel.handling_locally, handling_locally(Props, 'undefined')}
+      ,{#channel.import_moh, props:get_value(<<"variable_hold_music">>, Props) =:= 'undefined'}
       ,{#channel.interaction_id, props:get_value(<<?CALL_INTERACTION_ID>>, CCVs)}
+      ,{#channel.is_authorized, props:get_value(?GET_CCV(<<"Channel-Authorized">>), Props)}
       ,{#channel.is_loopback, kzd_freeswitch:is_loopback(Props)}
       ,{#channel.loopback_leg_name, kzd_freeswitch:loopback_leg_name(Props)}
       ,{#channel.loopback_other_leg, kzd_freeswitch:loopback_other_leg(Props)}
-      ,{#channel.callflow_id, props:get_value(<<"CallFlow-ID">>, CCVs)}
-      ,{#channel.cavs, CAVs}
+      ,{#channel.owner_id, props:get_value(<<"Owner-ID">>, CCVs)}
+      ,{#channel.precedence, kz_term:to_integer(props:get_value(<<"Precedence">>, CCVs, 5))}
+      ,{#channel.presence_id, props:get_value(<<"Channel-Presence-ID">>, CCVs, props:get_value(<<"variable_presence_id">>, Props))}
+      ,{#channel.profile, props:get_value(<<"variable_sofia_profile_name">>, Props)}
+      ,{#channel.realm, props:get_value(<<"Realm">>, CCVs, get_realm(Props))}
+      ,{#channel.reseller_billing, props:get_value(<<"Reseller-Billing">>, CCVs)}
+      ,{#channel.reseller_id, props:get_value(<<"Reseller-ID">>, CCVs)}
+      ,{#channel.resource_id, props:get_value(<<"Resource-ID">>, CCVs)}
+      ,{#channel.to_tag, props:get_value(<<"variable_sip_to_tag">>, Props)}
+      ,{#channel.username, props:get_value(<<"Username">>, CCVs, get_username(Props))}
```

Comment:
@jamesaimonetti i understand the `sort` but for me when reading code, it makes more sense to group by `affinity`., `from_tag` close to `to_tag`. but i'm ok with it.

Reply:
Yeah, i find myself hating that more and more :) I was trying to see if handling_locally was included in the update proplist, and I was just like, man, sort this list! Lol.